packages/backend/src/managers/playground.ts

```diff
@@ -241,10 +239,12 @@ export class PlayGroundManager {
     return Array.from(this.playgrounds.values());
   }
 
-  async sendQueriesState() {
-    await this.webview.postMessage({
+  sendQueriesState() {
```

Comment:
that said, I think methods should still have return type

probably we're missing another linting rule

Reply:
Resolved